Description: Fix compile time warning about redefinition of DUPLEX_UNKNOWN
Author: James Troup <james.troup@canonical.com>
--- a/nicstat.c
+++ b/nicstat.c
@@ -96,7 +96,9 @@ extern int optind, opterr, optopt;
 
 #ifdef OS_LINUX
 typedef __u8			duplex_t;
+#ifndef DUPLEX_UNKNOWN
 #define	DUPLEX_UNKNOWN		0xFF
+#endif /* DUPLEX_UNKNOWN */
 #else
 typedef uint32_t		duplex_t;
 #define	DUPLEX_UNKNOWN		0
